things/service.go

```diff
@@ -32,9 +32,6 @@ var (
 // Service specifies an API that must be fullfiled by the domain service
 // implementation, and all of its decorators (e.g. logging & metrics).
 type Service interface {
-	// AddThing adds new thing to the user identified by the provided key.
-	AddThing(context.Context, string, Thing) (Thing, error)
-
 	// CreateThings adds a list of things to the user identified by the provided key.
 	CreateThings(context.Context, string, []Thing) ([]Thing, error)
```

Comment:
I recommend that you use a variadic function here (with `...`).

Reply:
Yes agree, looks much nicer.

Reply:
Good idea